dashboard/app/views/shared/_course_wide_block.haml

```diff
@@ -1,4 +1,6 @@
--if id == 'hoc'
+- showState ||= false
```

Comment:
At least everywhere I know we include block partials, I've specified either true/false. They're in all_courses, hoc_courses, k5_courses, and dashboard/app/views/devise/sessions/new.html.haml